components/menu/test/custom-view.js

```diff
@@ -22,7 +23,7 @@ class CustomView extends HierarchicalViewMixin(LitElement) {
 					margin-top: 1rem;
 				}
 				:host a {
-					color: var(d2l-color-ferrite);
+					color: var(--d2l-color-ferrite);
```

Comment:
This was a legitimate error as we needed to put the two dashes in front of the var... why it was not caught before I'm unsure.